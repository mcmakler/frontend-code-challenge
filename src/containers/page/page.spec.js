import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PageContainer from './page';

describe('test page container', () => {
  it('should render page container', () => {
    const component = shallow(<PageContainer><div>page container</div></PageContainer>);

    expect(toJson(component)).toMatchSnapshot();
  });
});
