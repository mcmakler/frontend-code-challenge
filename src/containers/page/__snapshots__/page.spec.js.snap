// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test page container should render page container 1`] = `
<div
  className="full-height page"
>
  <Header />
  <div
    className="container auto-margin full-height"
  >
    <div
      className="row center-xs full-height"
    >
      <div
        className="col-xs-12 relative"
      >
        <Loader>
          <div>
            page container
          </div>
        </Loader>
      </div>
    </div>
  </div>
</div>
`;
